src/ICSharpCode.SharpZipLib/BZip2/BZip2InputStream.cs

```diff
@@ -711,10 +715,22 @@ cache misses.
 					unzftab[seqToUnseq[tmp]]++;
 					ll8[last] = seqToUnseq[tmp];
 
-					for (int j = nextSym - 1; j > 0; --j)
+					var j = nextSym - 1;
+
+#if !NETSTANDARD2_0 && !NETFRAMEWORK
```

Comment:
These should probably be inclusive instead?

Reply:
That was my first idea. But then I thought that one day you might start supporting e.g. .NET 5.0 Therefore I decided that it is better to exclude all frameworks that are currently supported (in the sense they are targeted during build) but do not support `System.Numerics.Vector` vectorization, judging that you can start supporting newer framework, but probably not an older one that is not currently supported. I can, however, change that if you want.

Reply:
Is there any value in defining a custom define in the build and then using that in the code instead of the TFM values?
e.g. 
```#if USE_VECTORS```
or some such? (just wondering if that would be nicer than checking multiple values, in case the same approach or logic were to be used elsewhere?)

Reply:
I'd go with the regular agile way, i.e. leave it as is now (since it's the only usage), but if another vectorization comes up, then define and use such symbol. It might also happen that only *vector-enabled* TFMs will be supported on that moment.